jonprl-mode.el

```diff
@@ -48,6 +48,23 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   ("<>" . ?★)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   ("base" . ?𝔻)
+   ("fix" . ?μ)
```

Comment:
Well, maybe once I start porting some more category theory stuff over… :)

I'll probably just remove the fix replacement for now. It's a short name and I can do something custom locally if I really want to change it.